src/apis/index.js

```diff
@@ -138,3 +137,9 @@ export function postSkillFeedback(payload) {
     device_type: 'Web Client',
   });
 }
+
+export function getDefaultMapData() {
+  return ajax.get('http://api.ipstack.com/134.201.250.155', {
+    accessKey: '31fe38d0ec112087bc85f47241a58387',
```

Comment:
This code block is hit when the server does not respond with lat/long data. In that case, this API is called to get the lat/long values and render the map. There are two problems with this. The way this is implemented makes the map component behave async, which renders a "map here" link instead of the map. @ananya-agrawal had opened an issue to fix the map issue so I told her to fix this deprecated API which I have replaced with a dummy API call with some random static IP. Should I take it up in this PR itself or should I let her fix this after this gets merged?

Reply:
@akshatnitd

Reply:
yes sir, I'll fix this ASAP.